README.md

```diff
@@ -83,78 +85,89 @@ CassieQ is fully encapsulated and only needs to know your cassandra information.
 the tracking tables for queues.
 
-## How does CassieQ work?
 
-Messages are put into a queue and can be read from the queue with a visiblity timeout. This means
-that if the message isn't acked within the visiblity timeout it becomes visible again.  Most other queue systems
-deal with this by detecting severed connections but since angenlahir is http based and connectionless we can't rely on that.
+## Admin panel
 
+The admin API is available at `host:8081/admin`
 
-CassieQ works with 3 pointers into a queue.
+## API
 
-- A reader bucket pointer
-- A repair bucket pointer
-- An invisiblity pointer
+We have bundled a java client to talk to a simple rest api. The api supports
 
-These three pointers will be discussed in each section
+- Queue create
+- Put a message
+- Get a message
+- Ack a message
 
-In order to scale and efficiently act as a queue we need to leverage cassandra partitioning capabilities.
-Queues are actually messages bucketized into a fixed size group called a bucket.   
-Each message is assigned a monotonically increasing id that maps itself into a bucket. 
+Getting a message gives you a pop reciept that encodes the message index AND its version. This means that you can prevent multiple ackers of a message
+and do conditional atomic actions based on that message version.
 
-For example, if the bucket is size 20 and you have id 21, that maps into bucket 1 (21/20).  
+## Authentication
 
-Messages are always put into the bucket they correlate to, regardless if previous buckets are full.
+CassieQ has three forms of authentication
+ 
+- Global account access via key
+- Endpoint signing by key
+- Granular access via permission claims by account
+ 
+### Claims
 
-The reader has a pointer to its active bucket (the reader buket pointer) and scans the bucket for unacked visible messages.  If the bucket is full
-it tombstones the bucket indicating that the bucket is closed for processing.  If the bucket is NOT full, but all messages
-in the bucket are consumed (or being processed) AND the monotonic pointer has already advanced to the next bucket,t he current 
-bucket is also tombstoned. This means no more messages will ever show up in the current bucket... sort of
+Claims by account is the preferred method. To generate a claim for a user, you can create a named account key for that group (such as "ui-members") and create
+permission based url params for them. This may look of the form:
 
-### Repairing delayed writes
+```
+?auth=puag&sig=NygRs9GBh9n_i2s7KTMof0us-RXm5nt3RnlWKb3N15A
+```
 
-There is a condition that you can have a delayed write. For example, assume you generate monotonic ids in this sequence:
+Which can be appended to any API url.  The auth is of a simple form (concatenation of single character based permissions) which in this scenario indicates
 
 ```
-Id 19
-Id 20
-Write 20
-Write 19
+Put (p)
+Update (u)
+Add (a)
+Get (g)
```

Comment:
Yeah, good catch. I'll update that and actually I should use the names that are provided from the list permissions Api to be consistent too